### DISCUSSION TITLE
Can't view website on Brave?

### DISCUSSION THREAD

**Discussion by iam-cult** [elrumo/macOS_Big_Sur_icons_replacements#1124] [Help with the website] [answered]
I just switched to Brave from Vivaldi, and from Firefox previously, and when I tried to visit the website, it said it was currently down for maintenance. However, when I visited the site on a different browser (Firefox), it worked normally. I've tried disabling all of my ad/tracker blocking, but the problem is still there,

**Comment by elrumo** [comment, marked as answer]
That was due to an issue with session tokens, it's now been fixed.
